alembic/versions/6fae83d65ee4_taxonomy.py

```diff
@@ -0,0 +1,4646 @@
+"""taxonomy
```

Comment:
is this script final? have you tested the most recent version?

Reply:
Yes, it's the final one. I rolled it back to the previous one. The foreign key of taxonomy in target sequence is taxonomy.id, not taxonomy.tax_id. I used this one to rebuild my local database. Special Taxonomy detail is https://github.com/VariantEffect/mavedb-api/issues/155. It's used to handle the taxonomy is NULL. Most of those score sets are unpublished or invalid.